### ISSUE TITLE
only python 2 package uploaded to pypi for 2.5.3 release

### ISSUE THREAD

**Issue by blinkseb** [mtth/hdfs#129]
Hello,

It looks like there was an issue when uploading version 2.5.3 to pypi, only Python 2 artefact was uploaded, as you can see here: https://pypi.org/simple/hdfs/

We are using pipenv for locking, and it resolves the constraint to v2.5.3, which fails to install with python 3:

```
Skipping link https://files.pythonhosted.org/packages/a3/fa/e68fa4b6c53e855aa480b7e7e0553ecac84af3250b6c1e04405bd625d6d5/hdfs-2.5.3-py2-none-any.whl#sha256=4c3c285dff6b6bcc9b4c4a67280d768cc35fb6af561508fc99550ff409aeabe0 (from https://pypi.org/simple/hdfs/); it is not compatible with this Python
```

Can you fix the issue?

Thanks!

**Comment by mtth** [comment]
Thanks for bringing this up. I'm not sure whether there's an easy way to add an artifact after the initial upload, I'll publish a new version to be safe.

**Comment by mtth** [comment]
Published `2.5.4`; `hdfs-2.5.4.tar.gz` available [here](https://pypi.org/simple/hdfs/). Please reopen if this doesn't fix the issue.

**Comment by blinkseb** [comment]
wha, awesome, thanks a lot for such a fast reaction! I can confirm everything is now working :smiley: 

Thanks again!
